src/amo/components/Routes/index.js

```diff
@@ -77,8 +76,6 @@ const Routes = ({ _config = config }: Props = {}) => (
 
     <Route exact path="/:lang/:application/addon/:slug/" component={Addon} />
 
-    <Route exact path="/:lang/:application/guides/:slug/" component={Guides} />
```

Comment:
I'll leave the discussion to the issue but maybe we shouldn't just 404? 301 to the homepage  🤷

Reply:
We've decided to just 404 these.